spark/src/main/scala/geotrellis/spark/RasterRDD.scala

```diff
@@ -23,103 +23,60 @@ import org.apache.spark.rdd._
 
 import scala.reflect.ClassTag
 
-class RasterRDD[K: ClassTag](val tileRdd: RDD[(K, Tile)], val metaData: RasterMetaData) extends RDD[(K, Tile)](tileRdd) {
+class RasterRDD[K: ClassTag, T: ClassTag](val tileRdd: RDD[(K, T)], val metaData: RasterMetaData) extends RDD[(K, T)](tileRdd) {
```

Comment:
I think that `RasterRDD` should be the `Tile` implementation, and not change the type signature. So something like 

```scala
class RasterRDD[K] extends GeoRDD[(K, Tile)](tileRdd)
class MultiBandRasterRDD[K] extends GeoRDD[(K, MultiBandTile)](tileRDD)
```
makes sense, it's less type code for the user to type out and won't break the current API.